### ISSUE TITLE
Generates useless .smithy.lsp.log files in all folders in the workspace

### ISSUE THREAD

**Issue by mcmasn-amzn** [smithy-lang/smithy-vscode#80]
I have found that in projects where I have many folders, if a single .smithy file exists, all workspace folders end up with a `.smithy.lsp.log` file in the root.

It's not a major problem to include `.smithy.lsp.log` in my `.gitignore`, but it seems like this file is mostly useless. Could this file be generated somewhere else like a temp folder, or suppressed unless users are trying to debug the Smithy extension?

**Comment by srchase** [comment]
I agree. I've cut an issue to the Smithy Language Server to make this opt-in: https://github.com/smithy-lang/smithy-language-server/issues/123

**Comment by mcmasn-amzn** [comment]
Thanks!
